mariadb/rootfs/etc/confd/conf.d/mariadb-server.toml

```diff
@@ -0,0 +1,7 @@
+[template]
+src = "mariadb-server.tmpl"
```

Comment:
Following convention, elsewhere in the repository the `src` should be named `mariadb-server.cnf.tmpl`.

Also rename the corresponding file in the template folder.

Reply:
Oh, same goes for this `.toml` file, it should become `mariadb-server.cnf.toml`